src/cases/containers/query-management-container/query-management-container.component.ts

```diff
@@ -26,7 +26,7 @@ export class QueryManagementContainerComponent implements OnInit {
       body: new FormControl(null, Validators.required),
       isHearingRelated: new FormControl(null, Validators.required),
       hearingDate: new FormControl(null),
-      attachments: new FormControl([])
+      attachments: new FormControl([] as Document[])
```

Comment:
@johnbenjamin-hmcts - can you check with @DavidJayakumar whether this is the correct type: there is `Document` and also `FormDocument`

Reply:
@Daniel-Lam I checked the base branch _feature/query-management_ and it is Document and not FormDocument. Might be that the changes are not yet merged to the base branch yet.